_styles/home.css

```diff
@@ -548,14 +548,23 @@ input:focus + .focus-reveal {
     width: 100%;
 }
 
-#slingshot-pager .button {
+#slingshot-pager .button { /*left here for posterity -p.shannon*/
```

Comment:
Eh, if it's unused, remove it.

Reply:
Yeah, we can always look at the history in Git anyway.

Reply:
Okay, didn't know if that would be rude. Thank you!